src/com/google/javascript/jscomp/deps/SortedDependencies.java

```diff
@@ -71,8 +71,12 @@
   }
 
   public static class MissingProvideException extends Exception {
-    MissingProvideException(String provide) {
+    public MissingProvideException(String provide) {
       super(provide);
     }
+
+    public MissingProvideException(String provide, Exception e) {
+      super(provide, e);
```

Comment:
Are we displaying this to the user? It might be nicer to have a message like `could not find some/file.js` instead of just `some/file.js`